docs/src/userguide/language_basics.rst

```diff
@@ -86,7 +86,7 @@ Moreover, C :keyword:`struct`, :keyword:`union` or :keyword:`enum` are supported
 
         .. literalinclude:: ../../examples/userguide/language_basics/struct_union_enum.py
 
-        .. NOTE:: Currently, Pure Python mode does not support enums.
+        .. note:: Currently, Pure Python mode does not support enums.
 
```

Comment:
It's bot the only one, there are other cases that i saw, it's better to replace all note entries in this document with `note`

Reply:
i just will find and suggest change, i think it'll be more useful

Reply:
This conversation should be resolved